package-examples/subpackages/wordpress/README.md

```diff
@@ -1,29 +1,29 @@
-wordpress
-==================================================
+# wordpress
 
 # NAME
 
-  wordpress
+wordpress
 
 # SYNOPSIS
```

Comment:
Remove NAME and SYNOPSIS as they don't make sense for a package example

Reply:
Done.